ursula_cli/shell.py

```diff
@@ -251,18 +378,23 @@ def main():
                         help='Test syntax for playbook')
     parser.add_argument('--ursula-debug', action='store_true',
                         help='Run this tool in debug mode')
+    parser.add_argument('--provisioner',
+                        help='The external provisioner to use vagrant,heat)',
+                        default=None)
```

Comment:
yeah, it should ...  this was easier to remember how to do on a plane with bad wifi.